[2025-09-16T00:22:54.731+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline_orchestration.train_model manual__2025-09-15T23:59:14.086476+00:00 [queued]>
[2025-09-16T00:22:54.819+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline_orchestration.train_model manual__2025-09-15T23:59:14.086476+00:00 [queued]>
[2025-09-16T00:22:54.822+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-09-16T00:22:55.170+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): train_model> on 2025-09-15 23:59:14.086476+00:00
[2025-09-16T00:22:55.198+0000] {standard_task_runner.py:60} INFO - Started process 613 to run task
[2025-09-16T00:22:55.224+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline_orchestration', 'train_model', 'manual__2025-09-15T23:59:14.086476+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/orchestration.py', '--cfg-path', '/tmp/tmpqrt0gc9w']
[2025-09-16T00:22:55.232+0000] {standard_task_runner.py:88} INFO - Job 53: Subtask train_model
[2025-09-16T00:22:55.536+0000] {task_command.py:423} INFO - Running <TaskInstance: ml_pipeline_orchestration.train_model manual__2025-09-15T23:59:14.086476+00:00 [running]> on host ff4ba757c7c6
[2025-09-16T00:22:56.320+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Jerr-williams' AIRFLOW_CTX_DAG_ID='ml_pipeline_orchestration' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-15T23:59:14.086476+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-15T23:59:14.086476+00:00'
[2025-09-16T00:22:58.782+0000] {docker.py:359} INFO - Starting docker container from image 972111245465383674563253/prediction-apps:latest
[2025-09-16T00:25:32.607+0000] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2025-09-16T00:25:45.267+0000] {process_utils.py:131} INFO - Sending 15 to group 613. PIDs of all processes in the group: [613]
[2025-09-16T00:25:46.259+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 613
[2025-09-16T00:25:59.146+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-09-16T00:26:31.035+0000] {docker.py:521} INFO - Stopping docker container
[2025-09-16T00:26:46.626+0000] {process_utils.py:149} WARNING - process psutil.Process(pid=613, name='airflow task ru', status='disk-sleep', started='00:22:54') did not respond to SIGTERM. Trying SIGKILL
[2025-09-16T00:26:49.346+0000] {process_utils.py:86} INFO - Sending the signal 9 to group 613
[2025-09-16T00:26:49.399+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=613, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='00:22:54') (613) terminated with exit code -9
[2025-09-16T00:26:49.620+0000] {standard_task_runner.py:175} ERROR - Job 53 was killed before it finished (likely due to running out of memory)
[2025-09-16T00:26:50.092+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 143
[2025-09-16T00:27:04.038+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
